tekton/predicates.go

```diff
@@ -39,8 +39,8 @@ func IntegrationOrBuildPipelineRunFinishedPredicate() predicate.Predicate {
 			return false
 		},
 		UpdateFunc: func(e event.UpdateEvent) bool {
-			return (IsIntegrationPipelineRun(e.ObjectNew) || IsBuildPipelineRun(e.ObjectNew)) &&
```

Comment:
The reason behind removing IsIntegrationPipelineRun is due to predicate??

Reply:
I separated predicates for Build and Integration pipeline pipelines, so now it's part of IntegrationPipelineRunPredicate()